HVSample/build.gradle

```diff
@@ -16,6 +16,10 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
         }
     }
+    packagingOptions {
+        exclude 'META-INF/LICENSE'
```

Comment:
Did we figure out what to do regarding these licenses? Are we allowed to check in this code with these excluded?